index.html

```diff
@@ -49,5 +47,183 @@ <h1 class="intro__heading">
         </div>
 
     </section>
+     
+    <section class="container__card">
+      <figure class="card">
+            <figcaption class="card__img--cont">
+              
+             
+  <img src="./images/SnapshootPortfolio.png" alt="image of a project" class="img__card">
+                <h2 class="desc__project">Tonic</h2>
+              <ul class="project__info--cont">
+                <li class="project__info--title">canopy</li>
+                
+                <li class="project__info--dot">
+                    <i class="fas fa-circle color__circle"></i>
+                </li>
+                <li class="project__info--author">Back End Dev</li>
+                 <li class="project__info--dot">
+                  <i class="fas fa-circle color__circle"></i>
+                 </li>
+                
+                <li class="project__info--date">2015</li>
+              </ul>
+            
+            <ul class="list__lng--btn">
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">html</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">css</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">javascript</button></li>
+            </ul>
+             <button type="button" class="link__btn--project">See Project</button>
+
+             </figcaption>
+    
+    </figure>
+  <figure class="card">
+            <figcaption class="card__img--cont">
+                <img src="./images/SnapshootPortfolio1.png" alt="image of a project" class="img__card">
+                <H2 class="desc__project">Multi-Post Stories</H2>
+              <ul class="project__info--cont">
+                <li class="project__info--title">canopy</li>
+                <li class="project__info--dot">
+                    <i class="fas fa-circle color__circle"></i>
+                </li>
+                <li class="project__info--author">Back End Dev</li>
+                 <li class="project__info--dot">
+                  <i class="fas fa-circle color__circle"></i>
+                 </li>
+                <li class="project__info--date">2015</li>
+              </ul>
+            <p class="project__desc--text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list__lng--btn">
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">html</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">css</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">javascript</button></li>
+            </ul>
+             <button type="button" class="link__btn--project">See Project</button>
```

Comment:
- [ ] Your buttons should also have stylings for `:active` and `:hover` effects. Like so:

```
.link__btn--project:active {
  ...
}

.link__btn--project:hover {
 ...
}
```